src/policy/policy.cpp

```diff
@@ -189,6 +189,11 @@ bool AreInputsStandard(const CTransaction& tx, const CCoinsViewCache& mapInputs)
             if (subscript.GetSigOpCount(true) > MAX_P2SH_SIGOPS) {
                 return false;
             }
+        } else if (whichType == TxoutType::TX_BARE_DEFAULT_CHECK_TEMPLATE_VERIFY_HASH) {
+            // after activation, only allow bare with no scriptsig.
+            // pre-activation disallowing enforced via discouraged logic in the
+            // interpreter.
+            if (tx.vin[i].scriptSig.size() != 0) return false;
```

Comment:
The standardness of bare basic OP_CTV transactions should probably be covered by a test - whatever I return for `TxoutType::TX_BARE_DEFAULT_CHECK_TEMPLATE_VERIFY_HASH`, tests don't fail.

Reply:
interesting let me look into this one. we very well might be missing a test vector (in tx_invalid.json) for this case.

to be clear, if the template matches on the script, but the scriptSig is nonzero it'd be nonstandard.


meta: does it make sense to just get rid of TxoutType::TX_BARE_DEFAULT_CHECK_TEMPLATE_VERIFY_HASH? We don't strictly need it in this PR, except to ensure that we are able to test CTV in bare script. That should still be testable to a degree even without a standard type...